source/_integrations/alexa.smart_home.markdown

```diff
@@ -40,9 +40,58 @@ Steps to Integrate an Amazon Alexa Smart Home Skill with Home Assistant:
 - [Configure the Smart Home Service Endpoint](#configure-the-smart-home-service-endpoint)
 - [Account Linking](#account-linking)
 - [Alexa Smart Home Component Configuration](#alexa-smart-home-component-configuration)
+  - [Alexa Locale](#alexa-locale)
+  - [Proactive Events](#proactive-events)
+  - [Configure Filter](#configure-filter)
+  - [Alexa Display Categories](#alexa-display-categories)
 - [Supported Platforms](#supported-platforms)
+  - [Alarm Control Panel](#alarm-control-panel)
+    - [Arming](#arming)
+    - [Disarming](#disarming)
+  - [Alert, Automation, Group, Input Boolean](#alert-automation-group-input-boolean)
+  - [Binary Sensor](#binary-sensor)
+    - [Routines](#routines)
+    - [Doorbell Announcement](#doorbell-announcement)
+    - [Presence Detection with Binary Sensor](#presence-detection-with-binary-sensor)
+  - [Camera](#camera)
+  - [Climate](#climate)
+    - [Set Thermostat Temperature](#set-thermostat-temperature)
+    - [Thermostat Mode](#thermostat-mode)
+  - [Cover](#cover)
+    - [Open/Close/Raise/Lower](#opencloseraiselower)
+    - [Set Cover Position](#set-cover-position)
+    - [Set Cover Tilt](#set-cover-tilt)
+    - [Garage Doors](#garage-doors)
+  - [Fan](#fan)
+    - [Fan Speed](#fan-speed)
+    - [Fan Direction](#fan-direction)
+    - [Fan Oscillation](#fan-oscillation)
+  - [Image Processing](#image-processing)
+    - [Presence Detection Notification](#presence-detection-notification)
+  - [Input Number](#input-number)
+  - [Light](#light)
+    - [Brightness](#brightness)
+    - [Color Temperature](#color-temperature)
+    - [Color](#color)
+  - [Lock](#lock)
+    - [Unlocking](#unlocking)
+  - [Media Player](#media-player)
+    - [Change Channel](#change-channel)
+    - [Speaker Volume](#speaker-volume)
+    - [Equalizer Mode](#equalizer-mode)
+    - [Inputs](#inputs)
+    - [Playback State](#playback-state)
+    - [Seek](#seek)
+  - [Scene](#scene)
+  - [Script](#script)
+  - [Sensor](#sensor)
+  - [Switch](#switch)
+  - [Timer](#timer)
+  - [Vacuum](#vacuum)
 - [Alexa Web-Based App](#alexa-web-based-app)
 - [Troubleshooting](#troubleshooting)
+  - [Binary Sensor not available in Routine Trigger](#binary-sensor-not-available-in-routine-trigger)
+  - [Token Invalid and no Refresh Token Available](#token-invalid-and-no-refresh-token-available)
 - [Debugging](#debugging)
```

Comment:
Please revert this TOC change

Reply:
Fix it

Reply:
This has not been resolved. the whole change is till there, please revert this.